# Maximum joint-space acceleration, just for safety.
max_joint_acceleration: 3.0

# Maximum cartesian end-effector velocity
max_position_acceleration: 2.0
max_position_velocity: 0.5
max_rotation_velocity: 3.0

# Control frequency
arm_frequency: 20

# right arm velocity factor
feedback_velocity_factor: 0.0
feedforward_velocity_factor: 1.0

start_rotation_q:
   x: 0.211
   y: 0.639
   z: 0.684
   w: 0.283
start_position:
   x: 0.587
   y: -0.167
   z: 1.009

start_joints: [-0.568087290316, 0.347189730735, -1.4214391721, -1.48169799187, -0.435971979788, 0.0, 0.0]


group_name: right_arm

eef_link: rh_palm

joint_names:
    - r_shoulder_pan_joint
    - r_shoulder_lift_joint
    - r_upper_arm_roll_joint
    - r_elbow_flex_joint
    - r_forearm_roll_joint
    - rh_WRJ2
    - rh_WRJ1

teleop_signal:
  start: 2
  stop: 4

arm_controller_name: r_arm_controller

goal_topic: /right/goal_wrist_transform_stamp
